projboard/migrations/data/articles_content.py

```diff
@@ -0,0 +1,36 @@
+# Generated by chat-gpt
```

Comment:
remove this comment . (;

Reply:
This is just a credit to the creators of the data